File: boj/Implemenation/15351/15351.cpp
#include <bits/stdc++.h>
using namespace std;
int n, score;
string s;
int main(){
  cin >> n;
  cin.ignore();
  while(n--){
    score = 0;
    getline(cin, s);
    for(auto c : s){
      if(c == ' ') continue;
      score += c - 'A' + 1;
    }
    if(score == 100) cout << "PERFECT LIFE\n";
    else cout << score << '\n';
  }
}

File: boj/Implemenation/15351/15351.rs
use std::io;

fn main(){
  let mut line = String::new();
  io::stdin().read_line(&mut line).expect("wrong io");
  let n = line.trim().parse::<i64>().unwrap();
  for _ in 0..n {
    let mut score = 0;
    let mut line = String::new();
    io::stdin().read_line(&mut line).expect("wrong io");
    let life = line.trim();
    for c in life.chars() {
      if c == ' ' {
        continue;
      }
      score += c as usize - 'A' as usize + 1;
    }
    if score == 100 {
      println!("PERFECT LIFE");
    }
    else {
      println!("{}", score);
    }
  }
}